packages/components/tsconfig.cjs.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
tsconfig.cjs.json extends the package's base tsconfig.json and only overrides the module and outDir settings.